readme.txt

```diff
@@ -70,6 +70,17 @@ In any case, passing the checks in this tool likely helps to achieve a smooth pl
 = 1.1.0 =
 
 * Enhancement - Changed the namespace of included checks.
+* Enhancement - Introduced severity levels for all errors and warnings.
+* Enhancement - New runtime check to warn about enqueued scripts that use neither `defer` nor `async`.
+* Enhancement - CLI: Support checking a plugin from a path or URL.
+* Enhancement - Added short descriptions and URLs for each check.
+* Enhancement - Improved messaging in check results.
+* Enhancement - Updated code obfuscation check with more accurate results.
+* Enhancement - Updated plugin review check to flag missing input sanitization.
+* Fix - Improve readme checks to exclude invalid files.
+* Fix - Only show edit link if files are actually editable.
+
+Input data must be Sanitized and Validated
```

Comment:
I'd make more clear the new checks added/updated

Reply:
Whoops, I wanted to remove this actually 😅